[imgui-sfml] include failure

**Host Environment**
- OS: Windows 10 / Ubuntu 20.04
- Compiler: MSVC 19.29.30137(x64) / g++ 9.3

**To Reproduce**
`./vcpkg install imgui-sfml`

Small snippet to reproduce the linker error.
```cpp
#include <SFML/Graphics/RenderWindow.hpp>
#include <imgui-SFML.h>
#include <imgui.h>

int main()
{
    sf::RenderWindow window(sf::VideoMode(640, 480), "Test window");
    ImGui::SFML::Init(window);
    ImGui::SFML::Shutdown();
}
```

(Also, manifest mode projects give the same errors.)

**Failure logs** 
```powershell
# MSVC classic mode project
1>------ Build started: Project: ConsoleApplication2, Configuration: Debug x64 ------
1>ConsoleApplication2.cpp
1>ImGui-SFML.lib(imgui-SFML.cpp.obj) : error LNK2019: unresolved external symbol "void __cdecl ImGui::Image(unsigned int,struct ImVec2 const &,struct ImVec2 const &,struct ImVec2 const &,struct ImVec4 const &,struct ImVec4 const &)" (?Image@ImGui@@YAXIAEBUImVec2@@00AEBUImVec4@@1@Z) referenced in function "void __cdecl ImGui::Image(class sf::Texture const &,class sf::Vector2<float> const &,class sf::Color const &,class sf::Color const &)" (?Image@ImGui@@YAXAEBVTexture@sf@@AEBV?$Vector2@M@3@AEBVColor@3@2@Z)
1>ImGui-SFML.lib(imgui-SFML.cpp.obj) : error LNK2019: unresolved external symbol "bool __cdecl ImGui::ImageButton(unsigned int,struct ImVec2 const &,struct ImVec2 const &,struct ImVec2 const &,int,struct ImVec4 const &,struct ImVec4 const &)" (?ImageButton@ImGui@@YA_NIAEBUImVec2@@00HAEBUImVec4@@1@Z) referenced in function "bool __cdecl ImGui::ImageButton(class sf::Texture const &,class sf::Vector2<float> const &,int,class sf::Color const &,class sf::Color const &)" (?ImageButton@ImGui@@YA_NAEBVTexture@sf@@AEBV?$Vector2@M@3@HAEBVColor@3@2@Z)
1>C:\Users\Home\source\repos\ConsoleApplication2\x64\Debug\ConsoleApplication2.exe : fatal error LNK1120: 2 unresolved externals
1>Done building project "ConsoleApplication2.vcxproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
```



**Additional context**
v2.3 port worked fine, but as soon as I updated vcpkg to use the v2.4 port, this error pops up.
Probably something is broken in the v2.4 port update(https://github.com/microsoft/vcpkg/pull/21460)?

--- a/ports/imgui-sfml/005-fix-imtextureid-define.patch
+++ b/ports/imgui-sfml/005-fix-imtextureid-define.patch
@@ -1,0 +1,9 @@
+diff --git a/imconfig-SFML.h b/imconfig-SFML.h
+index ddfe05d..bd7174e 100644
+--- a/imconfig-SFML.h
++++ b/imconfig-SFML.h
+@@ -24,4 +24,3 @@
+                          static_cast<sf::Uint8>(z * 255.f), static_cast<sf::Uint8>(w * 255.f));    \
+     }
+ 
+-#define ImTextureID unsigned int

--- a/ports/imgui-sfml/portfile.cmake
+++ b/ports/imgui-sfml/portfile.cmake
@@ -9,6 +9,7 @@ vcpkg_from_github(
     PATCHES
         0001-fix_find_package.patch
         004-fix-find-sfml.patch
+        005-fix-imtextureid-define.patch
 )
 
 vcpkg_cmake_configure(

--- a/ports/imgui-sfml/vcpkg.json
+++ b/ports/imgui-sfml/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "imgui-sfml",
   "version": "2.4",
+  "port-version": 1,
   "description": "ImGui binding for use with SFML",
   "homepage": "https://github.com/eliasdaler/imgui-sfml",
   "dependencies": [

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2842,7 +2842,7 @@
     },
     "imgui-sfml": {
       "baseline": "2.4",
-      "port-version": 0
+      "port-version": 1
     },
     "imguizmo": {
       "baseline": "1.83",

--- a/versions/i-/imgui-sfml.json
+++ b/versions/i-/imgui-sfml.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "5a40fbdee68d1f6157c8b5e0784fb701b290d0cc",
+      "version": "2.4",
+      "port-version": 1
+    },
+    {
       "git-tree": "c81a9d0362c06279990e889cb56f9ebf71bde380",
       "version": "2.4",
       "port-version": 0

